### PR TITLE
Adding files from scib

### PR DESCRIPTION
**Pull request by mumichae**
Scripts added:

- [x] `scripts/preprocessing.py`
- [x] `scripts/run_normalize.sh`
- [x] `scripts/sim_norm.py`
- [x]  `R/visualization/*`
- [x]  `notebooks/*`

Corresponding PRs:
- scib: https://github.com/theislab/scib/pull/217
- scib-pipeline: https://github.com/theislab/scib-pipeline/pull/1

### CODE REVIEW

**Review by LuckyMD — APPROVED** [review]
ideally add the R/visualization stuff already now. But fine to do it in stages as well.

**Comment by mumichae** [comment]
@martaint Is it safe to move all the `R/visualization` code here or are you still working on it?

**Comment by martaint** [comment]
I think it's safe! I'm not working on it :)

File: visualization/img/init [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (lazappi) [inline visualization/img/init:1]: Not sure where this blank file came from but can probably delete it.

**Review by lazappi — CHANGES_REQUESTED** [review]
Wow, that's a lot of notebooks! I assume somebody has already checked these are all still relevant?
* Do we need the `.ipynb_checkpoints` directories?

Should we move the data files in the `visualization/` directory to the top `data/` directory? At least those that are for the paper not just examples for the plotting functions (@martaint might be able to help with that).

**Comment by mumichae** [comment]
> Wow, that's a lot of notebooks! I assume somebody has already checked these are all still relevant?

We removed notebooks some time back ([PR](https://github.com/theislab/scib/pull/94)), so I think things should be OK. @LuckyMD @danielStrobl Could you double-check if there are any notebooks we don't need anymore?

> 
>     * Do we need the `.ipynb_checkpoints` directories?
> 

Good point, didn't realise that I was adding those. They should be removed and ignored now.

> 
> Should we move the data files in the `visualization/` directory to the top `data/` directory? At least those that are for the paper not just examples for the plotting functions (@martaint might be able to help with that).

I agree, that would be better. We just need to update the R scripts accordingly.

**Comment by LuckyMD** [comment]
Things I think might be removable:
1. The empty file @lazappi highlighted
2. Mouse atlas notebooks (@danielStrobl and @mumichae)
3. notebooks/Visualization_ATAC.ipynb from @kridsadakorn 
4. notebooks/silhouette_graph.ipynb, notebooks/trajectory_plots and notebooks/visualization.ipynb from @danielStrobl
5. visualization/atlas_heatmap.ipynb from @mumichae 

Would love to hear if you think these can be removed or if this is important for reproducibility. If important, then maybe we need a further folder.

**Review by kridsadakorn — COMMENTED** [review]
notebooks/Vitualisation_ATAC.ipynb can be deleted because the plots were replaced by Daniel's version.

PS. there is even a typo in the file name, poor me ...

**Comment by danielStrobl** [comment]
One thing I noticed is that the notebooks/visualisation.ipynb in master is not the most recent one, it's actually in the `recompute_cluster_metrics` branch

**Comment by LuckyMD** [comment]
Thanks for comments @danielStrobl. Could you say how you would name the folder (or in which existing folder) the notebooks to keep should be put? Just visualization?

**Comment by danielStrobl** [comment]
Yes, sounds good

**Comment by lazappi** [comment]
> One thing I noticed is that the notebooks/visualisation.ipynb in master is not the most recent one, it's actually in the `recompute_cluster_metrics` branch

@danielStrobl Can you open a separate PR for the latest version (or add it here)? Or open an issue I guess, just don't want to forget about this.

P.S. I think you have a pending invite to join this repo, I can resend if you need.

**Comment by danielStrobl** [comment]
Ok, I'll open a PR for that. That would be great if you could resend the invite.

**Comment by mumichae** [comment]
I agree that the heatmap and mouse atlas notebooks can be removed.

**Comment by mumichae** [comment]
I removed the heatmap plotting script (duplicate) and moved the preprocessing scripts to the [Masterinternship_2019](https://github.com/theislab/Masterinternship_2019) repository.

**Comment by LuckyMD** [comment]
also, new analysis notebook updates from scIB.

**Comment by lazappi** [comment]
Summarising things that have come up in comments:

**Done**

- [x] Resend invite to @danielStrobl
- [x] Remove `.ipynb_checkpoints` directories
- [x] Remove mouse atlas notebooks 

**Todo**

- [ ] Remove empty `visualization/img/init` file
- [ ] Move the visualisation data files to the `data/` directory (this probably means other changes to scripts so maybe can be a later PR)
- [ ] Remove `notebooks/Vitualisation_ATAC.ipynb`
- [ ] Remove `visualization/atlas_heatmap.ipynb`
- [ ] Remove `notebooks/silhouette_graph.ipynb`
- [ ] Update `notebooks/visualisation.ipynb` to most recent version
- [ ] Other notebook updates from **scIB**?

Anything that's incorrect or I missed?

**Comment by lazappi** [comment]
Also just noticed that each of the dataset notebook folders has an empty `README`. We should probably either add some info about the datasets or just delete them. @LuckyMD what do you think?

**Comment by LuckyMD** [comment]
Could you copy over some text from the dataset description in the paper to the README?

**Comment by lazappi** [comment]
> Could you copy over some text from the dataset description in the paper to the README?

That's definitely possible. Alternatively could describe what is in the notebooks (or both).

**Comment by LuckyMD** [comment]
Or maybe better: everyone could write a small blurb (3-4 sentences of what is in the dataset). I think that could even be from the SI sections.

**Comment by lazappi** [comment]
Did some of the minor things, see updated checklist https://github.com/theislab/scib-reproducibility/pull/4#issuecomment-820517623. Main things left to do are double checking notebooks are up to date (@danielStrobl can you help with that?) and deciding what to do with the READMEs.

**Comment by martaint** [comment]
@lazappi I added a file in scib (https://github.com/theislab/scib/pull/234) which should also be added here in the respective notebook folder. Could you take care of that? Don't want to mess with this PR :) thanks a lot!

**Review by lazappi — APPROVED** [review]

**Review by lazappi — APPROVED** [review]

**Pull request merged**